src/search.rs

```diff
@@ -785,7 +883,8 @@ mod tests {
                 value: S("dolor sit amet, consectetur adipiscing elit"),
                 kind: S("text"),
                 number: 10,
-                nested: Nested { child: S("second") }
+                nested: Nested { child: S("second") },
+                _vectors: Some(Vectors::from(&[2000.0])),
```

Comment:
FYI: the integration testcases are failing on the `:latest` meilisearch image as the following experimental feature is now removed

- https://github.com/meilisearch/meilisearch-rust/pull/638

Reply:
```suggestion
                _vectors: None,
```